packages/react/src/patterns/sub-patterns/Layout/Layout.js

```diff
@@ -65,7 +70,10 @@ function _updateChild(type, stickyOffset, children) {
         React.cloneElement(child, {
           className: classnames(
             child.props.className,
-            _types[type] && _types[type][i] ? _types[type][i] : `${prefix}--col`
+            _types[type] && _types[type][i]
+              ? _types[type][i]
+              : `${prefix}--col`,
+            border ? `${prefix}--layout-border` : ''
```

Comment:
Should this be applied to the row instead?

Reply:
I've changed this to be applied to the row.